tests/syft/lib/torch/tensor/tensor_remote_method_api_suite_test.py

```diff
@@ -217,15 +218,22 @@ def test_all_allowlisted_tensor_methods_work_remotely_on_all_types(
     # Step 5: Test to see whether this method and arguments combination is valid
     # in normal PyTorch. If it this is an invalid combination, abort the test
     try:
+        is_property = False
         # if this is a valid method for this type in torch
-        valid_torch_command = True
-        if args is not None:
-            target_result = target_op_method(*args)
+        if type(target_op_method).__name__ in ["builtin_function_or_method", "method"]:
+            valid_torch_command = True
```

Comment:
Q: Can't we move this outside ```if```. In this case we will remove the ```valid_torch_command = True``` in else

Reply:
Good spot! Thanks.